lib/ain-evm/src/evm.rs

```diff
@@ -512,6 +535,11 @@ impl EVMServices {
 
         self.filters.add_tx_to_filters(signed_tx.transaction.hash());
 
+        self.channel
+            .sender
+            .send(Notification::Transaction(signed_tx.transaction.hash()))
+            .map_err(|e| format_err!(e.to_string()))?;
+
```

Comment:
Likewise for here. At this point, it could be called on the miner or on connectblock when an incoming block is still be validated. Txs pushed into the queue at this point havent been executed yet on construct block and it is an incorrect event trigger to notify subscribers of the signed tx being minted.

Reply:
We should expose an FFI at the end of connect block that then iterates through the block and publishes the txs inside the block.

Reply:
Pending transactions should be obtained from the mempool instead. It should be inside the mempool pipeline, and we expose an FFI call for this

Reply:
Fixed: cbc5cab